fix!: stop provide Web API polyfills in components package (#976)
transferwise/neptune-web: packages/docs/pages/about/Setup.mdx

By default, apps come polyfilled. If you would like to use an unpolyfilled version because your app provides its own polyfills, see [Advanced Components Usage](#advanced-components-usage) below.

> <h2 id="polyfills-for-web-browser-api">Polyfills for Web / Browser API</h2>
> 
> We don't provide polyfills for <a href="https://developer.mozilla.org/en-US/docs/Web/API">Web API</a>, please provide it yourself in whichever way it best works for you.
> 
> Table of API that require some polyfills.

Reviewer: ```suggestion
<h2 id="polyfills-for-web-browser-api">Polyfills for IE</h2>

If you need to support Internet Explorer, you'll need some <a href="https://developer.mozilla.org/en-US/docs/Web/API">Web API</a> polyfills. An easy way to include them is using <a href="https://polyfill.io/v3/">Polyfill.io</a>

Components and their required polyfills
```

Author: polyfills are not only for IE, e.g polyfill for `Intl.Locale` is necessary for some previous versions of Safari, Firefox

polyfills necessary not only for components, some util functions too

<table className="docs-table table">
  <thead>
    <tr>
      <th>Item</th>
      <th>Require polyfill(s) for</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>

>         <DocLink href="components/context/Provider">Provider</DocLink> component

Reviewer: ```suggestion
        <DocLink href="components/context/Provider">Provider</DocLink>
```

Author: okey, I remove word "component" from here but again polyfills necessary not only for components

https://transferwise.github.io/neptune-web/branch/fix-intl.locale-polyfill/about/Setup#polyfills-for-web-browser-api

      </td>
      <td>
        <ul>
          <li>
            <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale">
              <code>Intl.Locale</code>
            </a>
          </li>
        </ul>
      </td>
    </tr>
    <tr>
      <td>

>         <DocLink href="components/content/Image">Image</DocLink> component

Reviewer: ```suggestion
        <DocLink href="components/content/Image">Image</DocLink>
```

      </td>
      <td>
        <ul>
          <li>
            <a href="https://developer.mozilla.org/en-US/docs/Web/API/IntersectionObserver/IntersectionObserver">
              <code>IntersectionObserver</code>
            </a>
          </li>
        </ul>
      </td>
    </tr>
    <tr>
      <td>

>         <DocLink href="components/inputs/DateInput">DateInput</DocLink> component

Reviewer: ```suggestion
        <DocLink href="components/inputs/DateInput">DateInput</DocLink>
```

      </td>
      <td>
        <ul>
          <li>
            <a href="https://developer.mozilla.org/en-US/docs/Web/API/Element/closest">
              <code>Element#closest</code>
            </a>
          </li>
        </ul>
      </td>
    </tr>
    <tr>
      <td>

>         <DocLink href="components/inputs/Upload">Upload</DocLink> component

Reviewer: ```suggestion
        <DocLink href="components/inputs/Upload">Upload</DocLink>
```

The components are exposed in two formats: UMD and ES. We also provide polyfilled and non-polyfilled versions.

<h3 id="polyfills-for-web-browser-api">Polyfills for Web / Browser API</h3>

> 

Author: https://transferwise.github.io/neptune-web/branch/fix-intl.locale-polyfill/about/Setup#polyfills-for-web-browser-api
